commercial/app/model/commercial/events/LiveEvent.scala

```diff
@@ -2,20 +2,27 @@ package model.commercial.events
 
 import model.commercial.events.Eventbrite._
 import org.joda.time.DateTime
-import play.api.libs.functional.syntax._
-import play.api.libs.json.{JsPath, Reads}
-
-case class LiveEventImage(eventId: String, mainImageUrl: String)
-object LiveEventImage{
-
-  implicit val liveEventImageReads: Reads[LiveEventImage] = (
-    (JsPath \ "id").read[String] and
-      (JsPath \ "mainImageUrl").read[String]
-    )(LiveEventImage.apply _)
-
+import play.api.data.validation.ValidationError
+import play.api.libs.json._
+
+case class LiveEventMembershipInfo(id: String,
+                                   url: String,
+                                   mainImageUrl: String)
+
+object LiveEventMembershipInfo {
+  implicit val format = Json.format[LiveEventMembershipInfo]
+
+  // based on play.api.libs.json.LowPriorityDefaultReads.traversableReads
+  implicit val formats = new Reads[Seq[LiveEventMembershipInfo]] {
+    override def reads(json: JsValue): JsResult[Seq[LiveEventMembershipInfo]] = {
+      json match {
+        case JsArray(jsValues) => JsSuccess(jsValues.flatMap(_.asOpt[LiveEventMembershipInfo]))
+        case _ => JsError(Seq(JsPath() -> Seq(ValidationError("error.expected.jsarray"))))
```

Comment:
Agreed, I was going off the example of https://github.com/guardian/frontend/blob/master/commercial/app/model/commercial/soulmates/Member.scala#L34

Ultimately, if we do enter that block then we were expecting to parse a JSON array but were not provided one - I suppose I can specify the expected content of the array i.e. LiveEventMembershipInfo